src/common/websocket/config.ts

```diff
@@ -32,6 +32,14 @@ export const domain_config = {
             hostname: 'eu.deriv.com',
             app_id: deriv_com_app_id,
         },
+        {
+            hostname: 'beta.deriv.com',
+            app_id: deriv_com_app_id,
+        },
+        {
+            hostname: 'beta-eu.deriv.com',
+            app_id: deriv_com_app_id,
+        },
```

Comment:
it's related to your task?

Reply:
above are not related to my task. it picked when I pulled from master when I was having issue with vercel.